src/Symfony/Component/Serializer/Context/Normalizer/AbstractNormalizerContextBuilder.php

```diff
@@ -164,4 +164,13 @@ public function withIgnoredAttributes(?array $ignoredAttributes): static
     {
         return $this->with(AbstractNormalizer::IGNORED_ATTRIBUTES, $ignoredAttributes);
     }
+
+    /**
+     * Configures flag to prevent fields that are nullable to receive a null
+     * fallback value when no parameters are provided for it.
+     */
+    public function withRequireAllProperties(?bool $requireAllProperties): static
```

Comment:
what about defaulting to `true`?

Reply:
I would love that, but that would be a breaking change, wouldn't it? Of am I misunderstanding you?

Reply:
I mean: the signature of the method should maybe be:
```suggestion
    public function withRequireAllProperties(?bool $requireAllProperties = true): static
```